### PR TITLE
Update gloo-fed install license

### PR DESCRIPTION
**Pull request by npolshakova**
# Description

Part of removing gloo-fed license to use gloo-ee license instead.

# Context

Make gloo-fed install command use same license as gloo-ee.

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: projects/gloo/cli/pkg/cmd/install/federation.go [hunk 1]
```diff
@@ -20,14 +20,12 @@ func glooFedCmd(opts *options.Options) *cobra.Command {
 		PreRun: setVerboseMode(opts),
 		RunE: func(cmd *cobra.Command, args []string) error {
 
+			opts.Install.HelmInstall = opts.Install.Federation.HelmInstall
+
 			extraValues := map[string]interface{}{
-				"license": map[string]interface{}{
-					"key": opts.Install.Federation.LicenseKey,
-				},
+				"license_key": opts.Install.LicenseKey,
```
Comment (ashleywang1) [inline projects/gloo/cli/pkg/cmd/install/federation.go:26]: There are a couple other places where this needs to be changed, e.g.:
projects/gloo/cli/pkg/cmd/demo/federation.go
in the glooctl demo federation command

Comment (ashleywang1) [reply projects/gloo/cli/pkg/cmd/install/federation.go:26]: Specifically, the line `licenseKey := opts.Install.Federation.LicenseKey` still exists there.

File: docs/content/installation/gloo_federation.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ With your kubectl context set to the admin cluster, run the following command:
 glooctl install federation --license-key <LICENSE_KEY>
 ```
 
-Make sure to change the placeholder `<LICENSE_KEY>` to the license key you have procured for Gloo Edge Federation.
+Make sure to change the placeholder `<LICENSE_KEY>` to the license key you have procured for Gloo Edge enterprise.
```
Comment (ashleywang1) [inline docs/content/installation/gloo_federation.md:32]: nit: capitalize

File: projects/gloo/cli/pkg/cmd/demo/federation.go [hunk 3]
```diff
@@ -33,7 +33,7 @@ func federation(opts *options.Options) *cobra.Command {
 		},
 	}
 	pflags := cmd.PersistentFlags()
-	flagutils.AddFederationDemoFlags(pflags, &opts.Install.Federation)
+	flagutils.AddFederationDemoFlags(pflags, &opts.Install)
```
Comment (ashleywang1) [inline projects/gloo/cli/pkg/cmd/demo/federation.go:36]: I think, with this, we might (?) be able to get rid of the Install.Federation struct entirely.

Comment (npolshakova) [reply projects/gloo/cli/pkg/cmd/demo/federation.go:36]: It looks like there's a conflict with the namespace name so we can't get rid of Install.Federation completely. :(

File: projects/gloo/cli/pkg/flagutils/install.go [hunk 4]
```diff
@@ -8,36 +8,36 @@ import (
...
+	set.BoolVar(&install.Federation.CreateNamespace, "create-namespace", true, "Create the namespace to install gloo fed into")
+	set.StringVar(&install.Federation.Namespace, "namespace", defaults.GlooFed, "namespace to install gloo fed into")
 	set.StringVar(&install.LicenseKey, "license-key", "", "License key to activate Gloo Fed features")
 }
 
-func AddFederationDemoFlags(set *pflag.FlagSet, install *options.Federation) {
+func AddFederationDemoFlags(set *pflag.FlagSet, install *options.Install) {
 	set.StringVar(&install.LicenseKey, "license-key", "", "License key to activate Gloo Fed features")
-	set.StringVar(&install.HelmChartOverride, "file", "", "Install Gloo Fed from this Helm chart archive file rather than from a release")
+	set.StringVar(&install.Federation.HelmChartOverride, "file", "", "Install Gloo Fed from this Helm chart archive file rather than from a release")
```
Comment (ashleywang1) [inline projects/gloo/cli/pkg/flagutils/install.go:40]: 👍 nice

**Comment by ashleywang1** [comment]
LGTM! Leaving open in case anyone else wants to take a look

**Review by kdorosh — APPROVED** [review]

**Pull request merged**